### PR TITLE
[12.0][ADD] attachment_azure with db operations

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Comment by yvaucher** [comment]
@max3903 can you rebase, attachment_azure was merged with this PR https://github.com/camptocamp/odoo-cloud-platform/pull/260

**Comment by max3903** [comment]
@yvaucher Done

File: attachment_azure/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+from odoo.http import request
+
+_logger = logging.getLogger(__name__)
+
+
+class Database(Database):
+    @http.route()
+    def drop(self, master_pwd, name):
+        res = super().drop(master_pwd, name)
+        delete = os.environ.get("AZURE_DELETE_ON_DBDROP", False)
```
Comment (yvaucher) [inline attachment_azure/controllers/main.py:16]: Ok, that's nice to make this opt-in :+1:

**Review by yvaucher — APPROVED** [review]
Great!

**Comment by yvaucher** [comment]
@max3903 just realized you documented it in https://github.com/camptocamp/odoo-cloud-platform/pull/253/files

I'll backport the documentation changes.

**Comment by yvaucher** [comment]
Doc pushed here: https://github.com/camptocamp/odoo-cloud-platform/commit/d512d877054977cd467d2a746862e186a85480ca

**Pull request merged**